gnomad_qc/v5/sample_qc/relatedness.py

```diff
@@ -130,10 +146,18 @@ def main(args):
                 parquet_uri=get_cuking_input_path(test=test),
                 overwrite=overwrite,
             )
+        if args.create_cuking_relatedness_table:
+            logger.info("Converting cuKING outputs to Hail Table.")
+            check_resource_existence(
+                output_step_resources={"relatedness_ht": relatedness(test=test).path}
+            )
+            ht = convert_cuking_output_to_ht(get_cuking_output_path(test=test))
+            ht = ht.repartition(args.relatedness_n_partitions)
```

Comment:
```suggestion
            ht = ht.naive_coalesce(args.relatedness_n_partitions)
```
I don't think it matters too much here, but I thought it was better to avoid `repartition` due to the shuffle